ruby-bugs.csv

```diff
@@ -49,7 +49,7 @@
 9009,core,Wrong binding when tracing c-calls
 9010,build,./configure --prefix= cannot handle directories with spaces
 9016,win encoding,String#encoding is lying?
-9045,lib,URIに含まれるエンコードされたID/パスワードがデコードされない
+9045,lib uriencoding,URIに含まれるエンコードされたID/パスワードがデコードされない
```

Comment:
Just adding `uriencoding` to add more clarity, but it feels different from the rest of the tickets with `encoding` so adding the `uri` prefix.